includes/class-amp-story-post-type.php

```diff
@@ -282,6 +336,13 @@ public static function filter_block_editor_settings( $editor_settings, $post ) {
 		if ( self::POST_TYPE_SLUG === $post->post_type ) {
 			unset( $editor_settings['colors'] );
 		}
+		if ( get_current_screen()->is_block_editor && isset( $editor_settings['styles'] ) ) {
+			foreach ( $editor_settings['styles'] as $key => $style ) {
+				if ( isset( $style['baseURL'] ) && false !== strpos( $style['baseURL'], get_stylesheet_directory_uri() ) ) {
```

Comment:
What about theme styles that are enqueued for AMP Stories specifically? Should this check for the `deps` as well to not remove if that is the case?

Reply:
These filtered styles are added via [`add_editor_style()`](https://developer.wordpress.org/reference/functions/add_editor_style/) which only gets the stylesheet as the param, there are no dependencies to check.

I've added a whitelisting filter instead within ac852a3, thoughts?